src/pages/en/migrate.md

```diff
@@ -6,6 +6,25 @@ description: How to migrate your project to latest version of Astro.
 
 Until Astro reaches v1.0, we expect to make some breaking changes across minor versions (ex: `v0.1 -> v0.2`). This guide exists to help you migrate to the latest versions of Astro and keep your codebase up-to-date.
 
+## Migrate to v0.25
+
+### A new integration system
+
+The `renderers` config has been replaced by a new, official integration system! When you update to `v0.25.0`, you should see a notice on startup that will tell you how you can update your configuration file to move from the deprecated `renderers` config to the new `integrations` config, complete with an example code snippet for you to copy.
+
+Read our ["Using Integrations"](/en/guides/integrations-guide) guide for more details.
+
+### No more built-in renderers
+
+This is a big one for anyone who hasn't yet defined their own `renderers: [...]` configuration in their `astro.config.mjs` file. Astro no longer comes with a default set of renderers for React, Preact, Svelte, and Vue. Instead, you'll need to seperately install the integration for the framework of your choice. Read our ["Using Integrations"](/en/guides/integrations-guide) guide for a step-by-step walkthrough of adding a new integration to your project.
+
+Looking ahead to the future, we have already started work on a helpful `astro add NAME` command that will be able to add new integrations to your project with a single command. But until this, this is still is a bit of a manual, one-time process to update your configuration.
+
+<!-- TODO: Shiki default! -->
```

Comment:
@bholmesdev would love your help here!

Reply:
I'm not @bholmesdev but I will point out that I expect a **significant** portion of our existing users who don't even know we ever got the option to switch the default to Shiki... let alone made that switch already.

Whatever goes in the Migration docs should start from the perspective that someone PROBABLY has Prism as a default, and should describe what they can immediately do to revert if a) they liked their Prism, or b) if they just don't want to deal with it YET (ie just switch to `prism` in config if the new default is causing any problems or throwing you off). They should already have a Prism theme installed or linked to, because they would have needed it, so that should "just work" for them.

Reply:
Yup, as of right now it should just be "hey, if you want to keep using prism that's fine, just add/change this one line of config." Will leave to Ben to write it up